### DISCUSSION TITLE
AIUTO TELEGRAM

### DISCUSSION THREAD

**Discussion by francesco1879** [indomus/forum#2189] [HUB: Home Assistant]
Ciao, sono ancora alle prime armi, ma ho cercato e seguito le varie guide sulla confezione delle notifiche con telegram e non riesco ad attivare questo servizio! 
C'è qualcuno che possa aiutarmi?
Grazie

**Comment by snakuzzo** [comment]
Ciao, perdonami ma richieste di aiuto di questo tipo non hanno senso.
Innanzi tutto dovresti cercare sul forum prima di aprire nuove discussion (soprattutto così generiche senza dettagliare nulla).
Se cerchi "Telegram", troverai tante discussioni simili da cui trarre spunto per sistemare quello che non sei riuscito ancora a fare.

**Reply by snakuzzo** [reply]
Chiudo questa discussione
